.github/workflows/test.yml

```diff
@@ -263,11 +263,13 @@ jobs:
     timeout-minutes: 120
     strategy:
       matrix:
-        colima-version: ["v0.5.5"]
+        colima-version: ["v0.6.5"]
     steps:
     - uses: actions/checkout@v4
       with:
-        fetch-depth: 1
+        # fetch-depth is set to 0 to let `limactl --version` print semver-ish version
+        fetch-depth: 0
+        ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
@AkihiroSuda 
Is there a reason for using the PR head commit instead of the PR merge commit? The code being tested might differ from what it will be after the merge, which seems problematic.

The intention here is not clear from reading the PR.
I am currently trying on a PR to consolidate build jobs per platform (https://github.com/lima-vm/lima/pull/2532), and I'm unsure if it's acceptable to default to using the merge commit. Could you clarify?

Reply:
From my memory `limactl --version` didn't show the correct version without this, but I could be wrong.
Feel free to remove this in that case.

Reply:
Thank you for your response. Based on my testing, `git describe` returns the expected version string format with just `fetch-depth: 0`, so it doesn’t seem necessary to set `ref:`.